.github/workflows/ci.yml

```diff
@@ -4,6 +4,11 @@ on: [pull_request]
 
 jobs:
   tests:
+    env:
+      RUN_EVAL: ${{ !(github.actor == 'github-actions[bot]') }}
+      run: echo "$RUN_EVAL"
+    if: "!(github.actor == 'github-actions[bot]') "
```

Comment:
[question] Playing devil's advocate on this one a bit, should we run ci.yml on all commits? For example, say there's a commit from a bot one day using this `github-actions[bot]` actor bumping the version of some dependencies in various packages within the repo. We may want to ensure tests, etc. are run for those packages in this case?

Reply:
The plan here is to match on the user and the base commit message if I can ever figure out how to get it. You're looking at a partial broken implementation here.